HelloWorld_ST_Sensors/user/bt/hc05.cpp

```diff
@@ -0,0 +1,32 @@
+#include "hc05.hpp"
+
+namespace bt {
+
+void hc05::write(const char* data) {
+  while (*data != 0) {
+    bt_uart_.putc(*data);
+    data++;
+  }
+}
+
+void hc05::register_process_func(void (*process)(const char* newline)) {
+  process_func_ = process;
+}
+
+void hc05::interrupt_cb() {
+  int data = bt_uart_.getc();
+  BUF_[counter_] = data;
```

Comment:
Maybe use a more expressive name? The underscore is a little blink-and-you-miss-it

Reply:
Changed `BUF_` name to `read_buf_`